MyOnboarding/src/MyOnboardingApp.ApiServices/ApiServicesBootstrapper.cs

```diff
@@ -9,17 +10,26 @@
 
 namespace MyOnboardingApp.ApiServices
 {
-    public class ApiServicesBootstrapper: IBootstrapper
+    public class ApiServicesBootstrapper : IBootstrapper
     {
-        public void Register(IUnityContainer container) 
+        public IUnityContainer Register(IUnityContainer container)
             => container
-            .RegisterType<HttpRequestMessage>(
-                new HierarchicalLifetimeManager(), 
-                new InjectionFactory(GetHttpRequestMessage))
-            .RegisterType<IUrlLocator, ItemUrlLocator>(new HierarchicalLifetimeManager());
+                .RegisterType<HttpRequestMessage>(
+                    new HierarchicalLifetimeManager(),
+                    new InjectionFactory(GetHttpRequestMessage))
+                .RegisterType<IUrlLocator, ItemUrlLocator>(new HierarchicalLifetimeManager());
 
 
-        private static HttpRequestMessage GetHttpRequestMessage(IUnityContainer container) 
-            => (HttpRequestMessage) HttpContext.Current.Items["MS_HttpRequestMessage"];
+        public void ValidateConfiguration(IUnityContainer container)
+        {
+            if (!container.IsRegistered<IRoutesConfig>())
+            {
+                throw new InvalidOperationException($"Configuration of {nameof(ApiServicesBootstrapper)} is invalid, there must registered instance of {nameof(IRoutesConfig)}.");
```

Comment:
At least:
```suggestion
                throw new InvalidOperationException($"Configuration of {nameof(ApiServicesBootstrapper)} is invalid, there must be an instance of {nameof(IRoutesConfig)} registered.");
```

But preferably, something more informative:
```suggestion
                throw new InvalidOperationException($"An implementation of {nameof(IRoutesConfig)} has to be registered in the `nameof(IUnityContainer)` in order for {nameof(IUrlLocator)} implementation to work properly.");
```
* The `nameof(ApiServicesBootstrapper)` is not essential, since there is a stack trace in every exception where the origin if an exception can be seen (on the other hand, seeing this in a log (where stack trace might not be present) could help with identifying the cause of the error for non-developers)
* The `{nameof(IUrlLocator)}` brings additional context, helpful when answering questions like "why is this condition even here? Will it work if I disable validation?"
* The `nameof(IUnityContainer)` is mentioned to pinpoint the true nature of the problem even more precisely.

Please, fix this in all `Validation` methods…